CHARTER.md

```diff
@@ -27,21 +27,30 @@ The goal for the working group then is to build a persona library and supporting
 
 Annually, the working group should survey and identify when and how the community is using the persona library successfully to fine-tune marketing materials, improve user experience, or better understand the buyer's journey. The survey results and insights are to be shared with the community.
 
-### 1.3 DevRel Software and Tools
+### 1.3 Catalog of Software and Tools
 
-Developer Relations professionals must use a diverse set of software and tools to accomplish the mission of the role. The value of having a directory of software and tools is to support the discoverability of resources that help solve day-to-day operational challenges. The problems which a tool may solve can be varied including customer relationship management, ticketing, content authoring, video editing, communication, software development, etc. 
+Developer Relations professionals must use a diverse set of software and tools to accomplish the mission of the role. The value of having a catalog of software and tools is to support the discoverability of resources that help solve day-to-day operational challenges. The problems which a tool may solve can be varied including customer relationship management, ticketing, content authoring, video editing, communication, software development, etc. 
 
-The goal for the working group then is to build a directory that can:
+The goal for the working group then is to build a catalog that can:
 
 - define organizing principles for what constitutes a tool (problem-oriented, categorical)
-- manage the directory for where tools are discoverable (storage, search, presentation)
+- manage the catalog for where tools are discoverable (storage, search, presentation)
 - provide a fair and balanced representation of resources that may include competing commercial products
 - system for dispute resolution where a community member and tool-owner may have diverging perspectives
 
-Annually, the working group should survey the community to identify changes in tooling used by professionals in their marketing, sales, product, and engineering tasks.
+Annually, the working group should survey the community for feedback to identify changes in tooling used by professionals in their marketing, sales, product, and engineering tasks.
 
```

Comment:
Do we want to include some content here defining the process for getting a tool added to the Catalog? Does it need to be reviewed or approved by X number of people first etc?

Reply:
Does #36 address the review / approval process sufficiently? 

I don't think we'd need to differentiate by asset type, or I'd be curious if you thought there may be something that would be unique we should call it out.

Reply:
I'll go ahead and resolve that #36 addresses it @robearlam but if you want to discuss it more we can.